setup.py

```diff
@@ -10,7 +10,8 @@
     'collective.transmogrifier',
     'setuptools',
     'z3c.autoinclude',
-    'zc.recipe.egg'
+    'zc.recipe.egg',
+    'plone.app.z3cform >=0.7.7,!=1.0' # handle group.__name__ being None
```

Comment:
hm... that would be able to pull anything > 1.0, not?
i'd say, it should be 'plone.app.z3cform >=0.7.7,<=1.0'
but then, i can't install plone.app.widgets.